src/NeonCore.ts

```diff
@@ -343,13 +343,23 @@ class NeonCore {
           action: 'edit',
           editorAction: editorAction
         };
-        function handle (evt: MessageEvent): void {
+        function handle (this: NeonCore, evt: MessageEvent): void {
```

Comment:
Adding typing for `this` keyword. This doesn't actually add a new argument for the function, it's just for TypeScript to type `this` properly.